comms/dht/src/proto/google.protobuf.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this file required?

Reply:
It's generated by prost because we use google's namespace. Haven't found a way to prevent that - I'll gitignore it.